release/ray_release/byod/byod_locust.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+# This script is used to build an extra layer on top of the base anyscale/ray image 
+
+pip install locust==2.16.1
```

Comment:
why does this need to be installed separately? could you add a comment to explain?

and nit: maybe add an end of line?

Reply:
Resolved by https://github.com/ray-project/ray/pull/44392